Code/Framework/AzToolsFramework/Tests/PropertyIntCtrlCommonTests.h

```diff
@@ -125,15 +113,19 @@ namespace UnitTest
             auto& widget = m_widget;
             auto& handler = m_handler;
             QString tooltip;
-            std::string expected;
+            std::stringstream expected;
```

Comment:
It'd be nice to get rid of `std::stringstream`, `AZStd::string` could be used instead.

````
const AZStd::string expected = AZStd::string::format("[%d, %d]", minString.toStdString().c_str(), maxString.toStdString().c_str());
...
EXPECT_STREQ(tooltip.toStdString().c_str(), expected.c_str());
````

Same for `HandlerMinMaxLessLimit_ModifyHandler_ExpectSuccessAndValidLessLimitToolTipString` test.